tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py{26,27,33,34,py,py3}-ripozo{release,git}
```

Comment:
Let's put a comment in here about what was removed py3 and why. We'll fork a card to ensure we re-add this in the future.